# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatform.h
# Commit: bf0dd96b4423
# Full Hash: bf0dd96b442327acd6df0db0f80886a53a4cec52
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 15:40:28
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 7 - Route COLRv1 fonts on macOS through our COLR-rendering path; only COLRv0 can use the native Core Text support. r=gfx-reviewers,lsalzman
#   
#   Depends on D153880
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152162
# ==============================================================================

diff -r d0a5d11277bb -r bf0dd96b4423 gfx/thebes/gfxPlatform.h
--- a/gfx/thebes/gfxPlatform.h	Wed Aug 17 10:59:35 2022 +0000
+++ b/gfx/thebes/gfxPlatform.h	Wed Aug 17 10:59:35 2022 +0000
@@ -782,8 +782,6 @@
 
   static bool HasVariationFontSupport();
 
-  bool HasNativeColrFontSupport() const { return mHasNativeColrFontSupport; }
-
   // you probably want to use gfxVars::UseWebRender() instead of this
   static bool WebRenderPrefEnabled();
   // you probably want to use gfxVars::UseWebRender() instead of this
@@ -924,10 +922,6 @@
   // Whether the platform supports rendering OpenType font variations
   static std::atomic<int8_t> sHasVariationFontSupport;
 
-  // Whether the platform font APIs have native support for COLR fonts.
-  // Set to true during initialization on platforms that implement this.
-  bool mHasNativeColrFontSupport = false;
-
   // The global vsync dispatcher. Only non-null in the parent process.
   // Its underlying VsyncSource is either mGlobalHardwareVsyncSource
   // or mSoftwareVsyncSource.